README.md

```diff
@@ -266,6 +266,13 @@ runner.run(collection, { /* options */ }, function(err, run) {
             //      -- Processed template
             //      processedTemplate: <String>
             //  }
+
+            // result: undefined or object containing following properties
```

Comment:
When is `result` undefined?

Reply:
When none of its property is available. Currently it has only one property `isSkipped`